frontend/src/features/SideWindow/MissionForm/types.ts

```diff
@@ -23,15 +24,59 @@ export type PartialGroundControl = Partial<GroundControl> & {
 }
 
 type SeaControl = {
+  compliance: {
+    ais: string
+    customInfractions: Array<{
+      natinfs: string[]
+      note: string
+      type: string
+    }>
+    jdp: string
+    permissions: string
+    vms: string
+  }
+  deviceInfractions: Array<{
+    declaredNetting: number
+    deviceName: string
+    isNettingUnmeasured: boolean
+    isOmegaGaugeMissing: boolean
+    measuredNetting: number
+    otmNote: string
+    pingersState: string
+  }>
+  hasNoOnboardSpecy: boolean
+  isOmegaGaugeMissing: boolean
+  specyInfractions: Array<{
+    specyName: string
+  }>
   startDate: Date
+  tideFishingZones: string[]
+  tideFleetSegments: string[]
   type: MissionType.SEA
 }
-export type PartialSeaControl = Partial<SeaControl> & {
+export type PartialSeaControl = Omit<
+  DeepPartial<SeaControl>,
+  'compliance' | 'deviceInfractions' | 'specyInfractions' | 'tideFishingZones' | 'tideFleetSegments'
+> & {
+  compliance: Partial<SeaControl['compliance']> & {
+    customInfractions: Array<Partial<SeaControl['compliance']['customInfractions'][0]>>
+  }
+  deviceInfractions: Array<Partial<SeaControl['deviceInfractions'][0]>>
+  specyInfractions: Array<
+    Partial<
+      Omit<SeaControl['specyInfractions'][0], 'specyName'> & {
+        specyName: string
+      }
+    >
+  >
   startDate: Date
+  tideFishingZones: string[]
```

Comment:
tide is not really used for a fishing voyage, I think we can use `faoZones` (because these zones are FAO codes)